website/blog/2024-09-19-bootc-microshift.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Creating a MicroShift bootable image with Podman Desktop
+description: Learn how to create a bootable image featuring MicroShift, using BootC and Podman Desktop.
+authors: [cdrage]
+tags: [podman-desktop, podman, rhel, bootc, extensions]
+hide_table_of_contents: false
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+![banner](img/bootc-microshift/redhat_selkie.png)
+
+If you're unfamiliar with [Image Mode for RHEL](https://developers.redhat.com/products/rhel-image-mode/overview), it offers an impressive method for deploying applications directly to bare metal from either a single Containerfile or a pre-existing bootc-supported image.
+
+A "bootable" image, known as a BootC container image, allows you to use a simple container image to create a full bootable operating system, whether it's a `raw` virtual machine image or an `iso` for USB installation!
+
+This capability is ideal for a variety of uses, from a simple HTTP server to an OS powering a full-stack application.
+
+In this tutorial, we'll deploy an OpenShift derivative called [MicroShift](https://www.redhat.com/en/topics/edge-computing/microshift), an edge-optimized version of OpenShift designed for single-node setups. Think of it as a compact version of OpenShift!
+
+This entire process is carried out using a single Containerfile (or Dockerfile).
+
+## Requirements
+
+Before starting the tutorial, ensure you have:
+
+- An active Red Hat account
```

Comment:
Would be good to have a link to registration form for those who does not have one.

Reply:
Maybe link to https://developers.redhat.com/articles/2024/05/07/podman-desktop-red-hat-developer-subscription?